File: python/json-io.py
import json

#input
file = open("../example.json")

text = file.read()

read_object = json.loads(text)
for key in read_object:
    print(key, read_object[key])

file.close()

#output
file = open("example-write.json",'w')

obj_to_write = { "foo":"bar", "baz":7, "qux":[1,2,3] }

file.write(json.dumps(obj_to_write, indent=4, separators=(',',':')))
file.close()

File: js/jsonio.js
text = '{ \
    "number":7, \
    "array":[ \
        1, \
        2, \
        3 \
    ], \
    "name":"foo" \
}'

console.log(JSON.parse(text));

output = document.getElementById("JSONtext");

objToWrite = {foo:"bar", baz:7, qux:[1,2,3]};

console.log(JSON.stringify(objToWrite));

output.innerHTML = JSON.stringify(objToWrite);
